boards/x86/qemu_x86/qemu_x86.dts

```diff
@@ -132,6 +132,10 @@
 	status = "okay";
 };
 
+&rtc {
```

Comment:
This could likely be moved to the RTC testsuite in a board overlay, since it is causing unrelated things to fail, maybe with a comment saying why it's necessary.

Reply:
I will move it to the test suite, and make sure the RTC tests are built